lib/spoom/snapshot.rb

```diff
@@ -23,6 +23,9 @@ def self.snapshot(path: '.')
       snapshot.calls_typed = metrics.fetch("types.input.sends.typed", 0)
       snapshot.calls_untyped = metrics.fetch("types.input.sends.total", 0) - snapshot.calls_typed
 
+      snapshot.duration += metrics.fetch("run.utilization.system_time.us", 0)
+      snapshot.duration += metrics.fetch("run.utilization.user_time.us", 0)
```

Comment:
I just wanted to point out that if the program runs on multiple CPUs `sys` + `user` time won't be equal to program duration. I don't know enough Ruby internals and if parts of the program will run in parallel with the GIL.

Reply:
I agree and it's not to be used as a benchmark tool.

The idea is more to spot big changes in typechecking time:

![image](https://user-images.githubusercontent.com/583144/95602035-5d92ce00-0a22-11eb-8343-7db6f070e2c3.png)